app/components/tag_component.rb

```diff
@@ -10,20 +10,20 @@ def initialize(text:, type:)
 
   def css_classes(type)
     tag_css_class = case type
-                    when :danger
-                      'app-tag--danger'
-                    when :secondary
-                      'app-tag--secondary'
-                    when :info
-                      'app-tag--info'
-                    when :info_unfilled
-                      'app-tag--info-unfilled'
-                    when :warning
-                      'app-tag--warning'
-                    when :primary_unfilled
-                      'app-tag--primary-unfilled'
+                    when :red
+                      'app-tag--red'
+                    when :grey
+                      'app-tag--grey'
+                    when :turquoise
+                      'app-tag--turquoise'
+                    when :green
+                      'app-tag--green'
+                    when :orange
+                      'app-tag--orange'
+                    when :purple
+                      'app-tag--purple'
                     else
-                      'app-tag--primary'
+                      'app-tag--blue'
                     end
```

Comment:
Since we're using colour names in the Perhaps we can simplify this by replacing everything with `app-tag--#{color}`.

Reply:
Thanks @tijmenb.

I was also thinking that the else statement should be killed. I think if you want it to be ‘blue’ then pass that in.

Then the mapping is more explicit further up the stream. What d'ya think?

Reply:
Yes great idea, go for it.